tests/test-definitions.sh

```diff
@@ -99,9 +99,8 @@ function verify_cuda_installation {
     check_exit_code "NVIDIA Peer memory module is inserted" "NVIDIA Peer memory module is not inserted!"
 
     # Verify if CUDA is installed
-    # re-enable this after testing
```

Comment:
Why was this commented before?

Reply:
oh, I need to recomment that out. It works ion VMs when I test, but fails in the pipeline for some reason. says that nvcc isn't a known command